src/protocol/mod.rs

```diff
@@ -38,6 +38,8 @@ pub enum IPAProtocolStep {
 #[derive(Copy, Clone, PartialEq, Eq, Hash)]
 pub enum SortStep {
     BitPermutations,
+    #[cfg(test)]
+    TestReveal,
```

Comment:
Not sure if this is an appropriate use of test config. But it seems to be working